Tries to create label when label already exists

Hi! I've been testing out different action for syncing labels in a repo and your action lines up perfectly with what we need. 

I keep running into an issue where the action doesn't recognize one of my labels right away. When running the action then it attempts to create the label then fails saying a label with that name already exists:

```
🎨 Creating 'wontfix' label with color 'ffffff' and desc 'This will not be worked on'
##[error]Validation Failed: {"resource":"Label","code":"already_exists","field":"name"}
```

The label is listed under my labels with the same fields above. All of the other labels are properly being skipped over, but I'm unsure why this one isn't?

I also tried using the `from_name:` with `wontfix` and it says the label can't be found in order to rename it. 

Let me know if there's any more information you need to help with this or if you have any guidance/things to try to fix this issue! Thanks!

Here's the log of Running the labeler:

```
⚠️ Skipping update for 'bug' label. Same color and description
⚠️ Skipping rename 'duplicate' label to 'duplicate_update'. Already exists
⚠️ Skipping update for 'enhancement' label. Same color and description
⚠️ Skipping update for 'good first issue' label. Same color and description
⚠️ Skipping update for 'help wanted' label. Same color and description
⚠️ Skipping update for 'invalid' label. Same color and description
⚠️ Skipping update for '' label. Same color and description
⚠️ Skipping update for '' label. Same color and description
⚠️ Skipping update for '' label. Same color and description
⚠️ Skipping update for '' label. Same color and description
⚠️ Skipping update for '' label. Same color and description
⚠️ Skipping update for '' label. Same color and description
⚠️ Skipping update for '' label. Same color and description
⚠️ Skipping update for '' label. Same color and description
⚠️ Skipping update for '' label. Same color and description
⚠️ Skipping update for '' label. Same color and description
⚠️ Skipping update for '' label. Same color and description
⚠️ Skipping update for '' label. Same color and description
⚠️ Skipping update for '' label. Same color and description
⚠️ Skipping update for '' label. Same color and description
⚠️ Skipping update for '' label. Same color and description
⚠️ Skipping update for '' label. Same color and description
⚠️ Skipping update for '' label. Same color and description
⚠️ Skipping update for '' label. Same color and description
⚠️ Skipping update for '' label. Same color and description
🎨 Creating 'wontfix' label with color 'ddc261' and desc 'This will not be worked on'
##[error]Validation Failed: {"resource":"Label","code":"already_exists","field":"name"}
🔫 Deleting 'asldkf'
```

(removed the sensitive label names)

file of labels found in '.github/labels/config.yml:
```
- name: "HI THERE"
  color: "d8bcf4"
  description: "label added by the labeler"
- name: "Epic"
  color: "d8bcf4"
  description: ""
- name: "stale"
  color: "d8bcf4"
  description: "This PR/issue is stale and will be deleted in 3 days if the label isn't removed"
- name: "never stale"
  color: "d8bcf4"
  description: "Never mark this PR/issue as stale"
- name: "bug"
  color: "d73a4a"
  description: "Something isn't working"
- name: "duplicate_update"
  color: "cfd3d7"
  description: "This issue or pull request already exists"
  from_name: "duplicate"
- name: "enhancement"
  color: "a2eeef"
  description: "New feature or request"
- name: "good first issue"
  color: "7057ff"
  description: "Good for newcomers"
- name: "help wanted"
  color: "008672"
  description: "Extra attention is needed"
- name: "invalid"
  color: "e4e669"
  description: "This doesn't seem right"
- name: "mp-rna"
  color: "29af1f"
  description: ""
- name: ""
  color: "bfd4f2"
  description: ""
- name: "question"
  color: "d876e3"
  description: "Further information is requested"
- name: ""
  color: "b7f7a0"
  description: ""
- name: ""
  color: "0052cc"
  description: ""
- name: ""
  color: "84ed84"
  description: ""
- name: ""
  color: "6fea54"
  description: ""
- name: ""
  color: "d8cb0f"
  description: ""
- name: ""
  color: "84ed84"
  description: ""
- name: ""
  color: "82ED88"
  description: ""
- name: ""
  color: "42cc0c"
  description: ""
- name: ""
  color: "84ed84"
  description: ""
- name: ""
  color: "84ed84"
  description: ""
- name: ""
  color: "84ed84"
  description: ""
- name: ""
  color: "84ed84"
  description: ""
- name: ""
  color: "84ed84"
  description: ""
- name: ""
  color: "84ed84"
  description: ""
- name: ""
  color: "88e579"
  description: ""
- name: ""
  color: "ddc261"
  description: ""
- name: "wontfix"
  color: "ddc261"
  description: "This will not be worked on"
```
and the workflow file:

```
name: Sync labels

on:
  push:
    branches: 
      - master
    paths:
      -.github/labels/config.yml
  label:
    types: [created, edited, deleted]

jobs:
  labeler:
    runs-on: ubuntu-latest
    steps:
      -
        name: Checkout
        uses: actions/checkout@v1
      -
        name: Run Labeler
        if: success()
        uses: crazy-max/ghaction-github-labeler@v1
        with:
          yaml_file: .github/labels/config.yml
          skip_delete: false
          dry_run: false
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

the action was triggered when I attempted to add a new label using the github UI

Did I put it in the workflow file correctly?/how you're suggesting? :

```
name: Sync labels

on:
  push:
    branches: 
      - master
    paths:
      - .github/labels/config.yml
  label:
    types: [created, edited, deleted]

jobs:
  labeler:
    runs-on: ubuntu-latest
    steps:
      -
        name: Checkout
        uses: actions/checkout@v1
      -
        name: Run Labeler
        if: success()
        uses: crazy-max/ghaction-github-labeler@labels-paginate
        with:
          yaml_file: .github/labels/config.yml
          skip_delete: false
          dry_run: false
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
  ```

If so, I'm getting this error:

`##[error]Could not find file '/home/runner/work/_actions/_temp_acd8bcaf-d6d6-4880-bf03-f38e348b020b/_staging/crazy-max-ghaction-github-labeler-b8a8a09/node_modules/.bin/esparse'.`

sorry for the delay. there was a pretty big power outage

--- a/lib/main.js
+++ b/lib/main.js
@@ -152,9 +152,14 @@ function run() {
 }
 function getLiveLabels() {
     return __awaiter(this, void 0, void 0, function* () {
-        return (yield octokit.issues.listLabelsForRepo(Object.assign(Object.assign({}, github.context.repo), { mediaType: {
-                previews: ['symmetra']
-            } }))).data;
+        const res = yield octokit.paginate(octokit.issues.listLabelsForRepo.endpoint.merge(Object.assign({}, github.context.repo)));
+        return res.map(label => {
+            return {
+                name: label.name,
+                color: label.color,
+                description: label.description || ''
+            };
+        });
     });
 }
 function getFileLabels(yamlFile) {

--- a/src/main.ts
+++ b/src/main.ts
@@ -160,14 +160,18 @@ async function run() {
 }
 
 async function getLiveLabels(): Promise<Array<Label>> {
-  return (
-    await octokit.issues.listLabelsForRepo({
-      ...github.context.repo,
-      mediaType: {
-        previews: ['symmetra']
-      }
+  const res = await octokit.paginate(
+    octokit.issues.listLabelsForRepo.endpoint.merge({
+      ...github.context.repo
     })
-  ).data as Array<Label>;
+  );
+  return res.map(label => {
+    return {
+      name: label.name,
+      color: label.color,
+      description: label.description || ''
+    };
+  }) as Array<Label>;
 }
 
 async function getFileLabels(yamlFile: fs.PathLike): Promise<Array<Label>> {

